### PR TITLE
Port TLC and Ohmage to dapp V2

### PR DESCRIPTION
**Pull request by nichosystem**
# Description
What does this PR solve?

Fixes # (issue)

# Checklist
- [x] Code follows the style guide
- [x] I have performed a self-review of my own code
- [x] New and existing tests pass locally
- [x] This PR is targeting the correct branch

### CODE REVIEW

**Comment by nichosystem** [comment]
Deploy preview for _core_ ready!

✅ Preview
https://core-6ttumfpj3-templedao.vercel.app
https://pr-878-preview.templedao.link

Built with commit de5e92c428cb1f0f60250ec251ca2632c49c4564.
This pull request is being automatically deployed with [vercel-action](https://github.com/marketplace/actions/vercel-action)

**Review by marshall2112 — CHANGES_REQUESTED** [review]
A few small things but otherwise it all seems to work

File: apps/dapp/src/components/Pages/Core/DappPages/Borrow/TLC/TLCModal.tsx [hunk 1]
```diff
@@ -4,11 +4,7 @@ import { useEffect, useState } from 'react';
 import { ZERO, fromAtto } from 'utils/bigNumber';
 import { TICKER_SYMBOL } from 'enums/ticker-symbol';
 import { useWallet } from 'providers/WalletProvider';
-import {
-  TempleLineOfCredit__factory,
-  ERC20__factory,
-  TreasuryReservesVault__factory,
-} from 'types/typechain';
+import { TempleLineOfCredit__factory, ERC20__factory, TreasuryReservesVault__factory } from 'types/typechain';
```
[the remark this answers is not in the document]
Comment (nichosystem) [reply apps/dapp/src/components/Pages/Core/DappPages/Borrow/TLC/TLCModal.tsx:7]: good call, deleted this file

File: apps/dapp/src/components/Pages/Core/DappPages/Borrow/index.tsx [hunk 2]
```diff
@@ -0,0 +1,625 @@
...
+    outputTokenBalance: ZERO,
+  });
+  const [activeScreen, setActiveScreen] = useState<'supply' | 'borrow'>('supply');
+  const [accountPosition, setAccountPosition] = useState<ITlcDataTypes.AccountPositionStructOutput>();
+  const [tlcInfo, setTlcInfo] = useState<TlcInfo>();
+  const [prices, setPrices] = useState<Prices>({ templePrice: 0, daiPrice: 0, tpi: 0 });
+
+  const getPrices = async () => {
+    const { data } = await fetchGenericSubgraph(
+      'https://api.thegraph.com/subgraphs/name/medariox/v2-mainnet',
```
[the remark this answers is not in the document]
Comment (nichosystem) [reply apps/dapp/src/components/Pages/Core/DappPages/Borrow/index.tsx:68]: fixed!

File: apps/dapp/src/components/Pages/Core/DappPages/OhmagePage.tsx [hunk 3]
```diff
@@ -118,42 +113,40 @@ export const OtcModal: React.FC<IProps> = ({ isOpen, onClose }) => {
...
+          <p>Exceeds {availableDai.toFixed(2)} DAI available for OTC</p>
+        ) : (
+          <p>You will receive {output.toLocaleString()} DAI</p>
+        )}
+        <TradeButton
+          onClick={() => {
+            if (insufficientAllowance) approve();
+            else swap();
+          }}
+          disabled={!signer || insufficientBalance || insufficientDaiAvailable}
```
[the remark this answers is not in the document]
Comment (nichosystem) [reply apps/dapp/src/components/Pages/Core/DappPages/OhmagePage.tsx:143]: fixed

File: apps/dapp/src/components/Pages/Core/DappPages/Borrow/TLC/Supply.tsx [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import leftCaret from 'assets/images/newui-images/leftCaret.svg';
-import { TradeButton } from '../Home';
-import { Input } from '../HomeInput';
+import { TradeButton } from '../../../NewUI/Home';
+import { Input } from '../../../NewUI/HomeInput';
```
[the remark this answers is not in the document]
Comment (nichosystem) [reply apps/dapp/src/components/Pages/Core/DappPages/Borrow/TLC/Supply.tsx:2]: Hm I'm having a tough time replicating. Will dm, can make it a separate PR for fixing too

**Pull request merged**